src/type/text2d.js

```diff
@@ -286,22 +279,22 @@ function text2d(p5, fn) {
         family += '-' + font.name.fontSubfamily;
       }
     }
-
-    if (typeof family !== 'string') {
-      throw Error('null font passed to textFont', font);
+    else if (typeof font === 'string') {
```

Comment:
Good catch for this one! Do we have a test yet anywhere that checks for a raw canvas string that includes the font size? If not, it could be a good thing to add to make sure we don't accidentally break it going forward